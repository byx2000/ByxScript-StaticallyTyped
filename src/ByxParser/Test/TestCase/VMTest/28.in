int g1 = -12;
int g2 = -6 + 30;
int g3 = +2 + 1;
int g4 = +6 - 30;
int g5 = (-2) + 1;
int g6 = (-6) + 30;
int g7 = (+2) + 1;
int g8 = (+6) - 30;
int g9 = -(6 + 3);
int g10 = +(6 + 3);

double g11 = -3.14;
double g12 = -2.7 + 6.63;
double g13 = -1-2-3;
double g14 = +35.7;

function void main()
{
	
}