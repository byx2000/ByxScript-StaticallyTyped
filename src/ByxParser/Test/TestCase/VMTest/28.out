i -12
i 24
i 3
i -24
i -1
i 24
i 3
i -24
i -9
i 9
d -3.14
d 3.93
d -6
d 35.7